apps/firelens-stability/collections/ecs-firelens-stability-tests/golden-path/cases/onepod-high-throughput.json

```diff
@@ -1,6 +1,6 @@
 {
     "definitions": {
-        "include_onepod": false,
+        "include_onepod": true,
         "kbps_throughput_tcp": 10000,
```

Comment:
I think we can reduce these to 5MB/s each, real world throughput is almost never this high I think given data Ive seen

Reply:
Updated all of these to 5mbps